src/generated/rpc/README.md

```diff
@@ -0,0 +1,3 @@
+# Google Cloud SDK for Rust - Google RPC Types
+
+Defines RPC types.
```

Comment:
Aside: is this generated and maintained by sidekick? If so, should it have a comment saying so?

Reply:
#493